_docs/sandbox-testing.md

```diff
@@ -14,9 +14,121 @@ To facilitate testing our sandbox environment has a few features that can help i
 
 You can use the `POST /transactions/[TRANSFERZERO_TRANSACTION_ID]/payout` endpoint with an empty body to force a funded, but not yet paid transaction to succeed. You can use this endpoint to test out if you are receiving successful transaction webhooks properly.
 
-# Forcing collections to succeed
+# Forcing transactions to pay in
 
-When using our [collections feature]({{ "/docs/additional-features/" | prepend: site.baseurl }}#collections-from-senders) you can force the collection to succeed. Please see the documentation of this feature in our [API specification guide](https://api.transferzero.com/documentation/#/Transactions/payin-transaction)
+You can use the `POST /transactions/[TRANSFERZERO_TRANSACTION_ID]/payin` endpoint with a `PayinMethodRequest` body to force a successful payin, for example to test out if you are receiving successful transaction webhooks properly. See the examples below using our SDKs
+
+{% capture data-raw %}
+```javascript
+POST /v1/transactions/e77f89fa-8371-496c-ae73-e56697fc08d8/payin
+{
+  "payin_method": {
+    "type": "GHS::Mobile"
+  }
+}
+```
+{% endcapture %}
+
+{% capture data-csharp %}
```

Comment:
Were these generated using the codegenerator?

Reply:
@sztupy  this has been updated